index.js

```diff
@@ -28,29 +29,36 @@ program
       if (err) throw new Error(err);
       var trimRes;
       var results = JSON.parse(body).list;
+      var resultsToDisplay = 3;
 
       console.log('='.repeat(consoleWidth()));
 
+      if(typeof process.argv[4] !== 'undefined'){
+        if(typeof Number(process.argv[4]) === 'number'){
+          resultsToDisplay = Number(process.argv[4]);
+        }
+      }
+
       if (results.length === 0) {
         console.log(chalk.red('No results were found, please try another phrase'));
       } else {
-        if (results.length > 3) {
-          trimRes = results.slice(0, 3);
+        if (results.length > resultsToDisplay) {
+          trimRes = results.slice(0, resultsToDisplay);
         } else {
           trimRes = results;
         }
         trimRes.forEach((result) => {
           if (typeof (result.definition) !== undefined) {
-            console.log('Word: ' + chalk.magenta(result.word));
-            console.log('Definition: ' + chalk.magenta(result.definition));
-            console.log('Score: ' + (result.thumbs_up - result.thumbs_down));
+            console.log(chalk.hex('#ffb3ff').bold('Word: ') + chalk.hex('#ffb3ff').bold(result.word));
+            console.log(chalk.hex('#ffb3ff').bold('Definition: ') + result.definition);
+            console.log(chalk.hex('#ffb3ff').bold('Score: ') + (result.thumbs_up - result.thumbs_down));
```

Comment:
Because we're using the same color so many times here, we can also set the color as a variable so if we wanted to change it , we're not changing it so many times.